ui/package.json

```diff
@@ -60,12 +60,13 @@
         "@types/react-router-dom": "^4.2.3",
         "@types/superagent": "^4.1.16",
         "@types/swagger-ui-react": "^4.11.0",
-        "babel-jest": "^29.5.0",
+        "@types/uuid": "^9.0.1",
+        "babel-jest": "^29.6.2",
         "babel-loader": "^8.3.0",
         "copy-webpack-plugin": "^5.1.2",
         "copyfiles": "^2.4.1",
         "file-loader": "^6.2.0",
-        "glob": "^9.3.2",
+        "glob": "^10.3.3",
```

Comment:
Hmm `glob` and `babel-jest` weren't changed in #11585 and you didn't mention them in your PR description -- possible incorrect merge conflict resolution?

I do see that #11585 already had them at these versions though, not sure if you intentionally bumped them?

Reply:
`babel-jest` looks like all dependabot updates and only patches and minors: #11307 and #11478

Reply:
Looks like `glob` was also all dependabot updates, including dependabot doing a breaking change (i.e. one of the reasons I disabled it for non-security in #12487). #10862, #10915, and several more until #11328

Reply:
Oh right, and then I removed `glob` in #11630 anyway as it was unused